Gopkg.toml

```diff
@@ -0,0 +1,102 @@
+# Gopkg.toml example
```

Comment:
can we have a comment at the top of this file explaining it's purpose and how developers use it.  Example:
This is the configuration file for the dep dependency management tool.
This file must be kept up to date with constraint entries for 3rd party libraries used by Scoot.